src/Tribe/Template.php

```diff
@@ -523,14 +523,18 @@ public function template( $name, $context = array(), $echo = true ) {
 		}
 
 		$name = $template_names[ $cache_name_key ];
+		$origin_folder_appendix = array_diff( $this->folder, $this->template_origin_base_folder );
 
 		if ( ! empty( $this->origin->template_namespace ) ) {
-			$namespace = array_merge( (array) $this->origin->template_namespace, $name );
+			$legacy_namespace = array_merge( (array) $this->origin->template_namespace, $name );
+			$namespace = array_merge( (array) $this->origin->template_namespace, $origin_folder_appendix, $name );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: 140a4cc8a90a34a614c073e56263e8dc</sub>